services/repository-managers/src/test/java/org/sagebionetworks/repo/manager/team/MembershipInvitationManagerImplTest.java

```diff
@@ -254,12 +258,29 @@ public void testCreateInvitationNotification() throws Exception {
 		MembershipInvtnSubmission mis = createMembershipInvtnSubmission(MIS_ID);
 		Team team = new Team();
 		team.setName("test team");
+		team.setId(TEAM_ID);
 		when(mockTeamDAO.get(TEAM_ID)).thenReturn(team);
-		MessageToUserAndBody result = membershipInvitationManagerImpl.createInvitationNotification(mis);
+		String acceptInvitationEndpoint = "https://synapse.org/#acceptInvitationEndpoint:";
+		String notificationUnsubscribeEndpoint = "https://synapse.org/#notificationUnsubscribeEndpoint:";
+		MessageToUserAndBody result = membershipInvitationManagerImpl.
+				createInvitationNotification(mis, acceptInvitationEndpoint, notificationUnsubscribeEndpoint);
 		assertEquals("you have been invited to join a team", result.getMetadata().getSubject());
 		assertEquals(Collections.singleton(MEMBER_PRINCIPAL_ID), result.getMetadata().getRecipients());
-		assertEquals(result.getBody(), "Hello,\r\nYou have been invited to join the team test team.  The inviter sends the following message:\r\n\r\nPlease join our team.\r\n\r\n  To view and accept the invitation, please visit this page: https://www.synapse.org/#!Team:123.\r\nSincerely,\r\nSynapse Administration\r\n\r\nTo turn off email notifications, please visit your settings page at https://www.synapse.org/#!Profile:999/settings\r\n", 
-				result.getBody());
+		String bodyNoWhiteSpace = result.getBody().replaceAll("\\s", "");
+		String s1 = "<html><body><p>Hello,</p><p>Youhavebeeninvitedtojointheteamtestteam.Theinvitersendsthefollowingmessage:<Blockquote>Pleasejoinourteam.</Blockquote>Tojointheteam,justclick<ahref=https://synapse.org/#acceptInvitationEndpoint:";
```

Comment:
Ideally, this would be fed by the template itself (so that modifications to the template would not break this test).